test/integration/root/inspec.yml

```diff
@@ -0,0 +1,8 @@
+name: root
+title: InSpec Profile
+maintainer: The Authors
+copyright: The Authors
+copyright_email: you@example.com
+license: All Rights Reserved
```

Comment:
not sure if these defaults can be deleted or not, but maybe make the license Apache-2.0 by default. I just noticed the other inspec.yml are also All Rights Reserved.